rosnav/model/custom_policy_utils/modules.py

```diff
@@ -0,0 +1,28 @@
+from torch import nn
+
+def linear(params):
+    return nn.Linear(params["in_features"], params["out_features"], params.get("bias", True))
+
+def relu(params):
+    return nn.ReLU(params.get("inplace", False))
+
+def conv1d(params):
+    return nn.Conv1d(params["in_channels"], params["out_channels"], params["kernel_size"], params.get["stride"], params.get["padding"], params.get["padding_mode"], params.get["dilation"], params.get["groups"], params.get["bias"])
+
+def conv2d(params):
+    return nn.Conv1d(params["in_channels"], params["out_channels"], params["kernel_size"], params.get["stride"], params.get["padding"], params.get["padding_mode"], params.get["dilation"], params.get["groups"], params.get["bias"])
+
+def conv3d(params):
+    return nn.Conv1d(params["in_channels"], params["out_channels"], params["kernel_size"], params.get["stride"], params.get["padding"], params.get["padding_mode"], params.get["dilation"], params.get["groups"], params.get["bias"])
+
+def tanh(params):
+    return nn.Tanh
+
+modules = {
+    "relu" : relu,
+    "tanh" : tanh,
+    "linear" : linear,
+    "conv1d" : conv1d,
+    "conv2d" : conv2d,
+    "conv3d" : conv3d,
+}
```

Comment:
Was du auch machen kannst um nicht so viele neue funktionen zu haben ist die funktionen direkt im dict aufzurufen:

```suggestion
def linear(params):
    return nn.Linear(params["in_features"], params["out_features"], params.get("bias", True))

def relu(params):
    return nn.ReLU(params.get("inplace", False))

...

modules = {
    "relu" :  nn.ReLU(params.get("inplace", False)),
    "linear" : nn.Linear(params["in_features"], params["out_features"], params.get("bias", True)),
    ...
}
```